src/boot/bootctl-status.c

```diff
@@ -243,10 +243,7 @@ static int enumerate_binaries(
                 /* Do not output this entry immediately, but store what should be printed in a state
                  * variable, because we only will know the tree glyph to print (branch or final edge) once we
                  * read one more entry */
-                if (r > 0)
-                        r = asprintf(previous, "/%s/%s (%s%s%s)", path, de->d_name, ansi_highlight(), v, ansi_normal());
-                else
-                        r = asprintf(previous, "/%s/%s", path, de->d_name);
```

Comment:
do we want to check for `-ESRCH` here and print anyway without version number? My gut feeling is no; as we only want to print info about files that are definitely owned by systemd.

Reply:
I'm not sure what the intended behaviour is. As before #27069 the printing would just break if a file not owned by systemd was found